packages/video_player/video_player_android/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## NEXT
```

Comment:
Please see the docs linked from the checklist on how to update a changelog containing NEXT.

Reply:
done.